transports/rabbitmq/index.md

```diff
@@ -17,6 +17,17 @@ Provides support for sending messages over [RabbitMQ](https://www.rabbitmq.com/)
 
 WARNING: The transport is not compatible with RabbitMQ broker version 3.3.X and below.
 
+## Transport facts
+
+|Feature                    |   |  
+|:---                       |---
+|Transactions |None, ReceiveOnly
+|Pub/Sub                    |Native
+|Timeouts                   |Native
+|Large message bodies       |via DataBus
```

Comment:
```suggestion
|Large message bodies       |Broker can handle arbitrary message size within available resources, very large messages via DataBus
```

Reply:
Maybe this is too many details anyway I wanted to add it for completeness reasons